bson/src/main/org/bson/BsonArray.java

```diff
@@ -29,9 +28,8 @@
  *
  * @since 3.0
  */
-public class BsonArray extends BsonValue implements List<BsonValue>, Serializable {
```

Comment:
BsonDocument now serializes itself as just BSON, and there is no way to represent a BSON array as BSON without a wrapping document.